File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int size = nums.length;
        int[] ans = new int[size * 2];

        for (int i = 0; i < ans.length; i++)
            ans[i] = nums[i % size];

        return ans;
    }
}

File: Data Structures & Algorithms/find-pivot-index/submission-0.py
class Solution:
    def pivotIndex(self, nums: List[int]) -> int:
        total = sum(nums)
        rsum = 0

        for i in range(len(nums)):
            if rsum * 2 + nums[i] == total:
                return i
            rsum += nums[i];
        
        return -1;